src/renderers/shaders/ShaderChunk/skinning_vertex.glsl

```diff
@@ -7,6 +7,7 @@
 	skinned += boneMatY * skinVertex * skinWeight.y;
 	skinned += boneMatZ * skinVertex * skinWeight.z;
 	skinned += boneMatW * skinVertex * skinWeight.w;
-	skinned  = bindMatrixInverse * skinned;
+
+	transformed = ( bindMatrixInverse * skinned ).xyz;
```

Comment:
Simply continue mutating `transformed` rather than creating a new variable. This removes the need to deal with the special case when skinning is enabled.